### PR TITLE
[SELS-TASK] BE-Get-Followers-API

### PR DESCRIPTION
**Pull request by ragg09-sunasterisk**
### Asana Link
https://app.asana.com/0/1203341829928748/1203341829928710/f

### Commands
```
cd backend
php artisan serve
cd frontend
npm start
```
### Pre-conditions
none

### Expected Output
- [X] be able to return selected user, and Followers (follower, followed) count

### Notes
- tested in postman

### Screenshots
image

### CODE REVIEW

File: be/app/Models/Follower.php [hunk 1]
```diff
@@ -23,4 +23,9 @@ public function log()
     {
         return $this->morphOne(Activity_log::class, 'loggale');
     }
+
+    public function follows($id)
+    {
+        return $this;
+    }
```
Comment (escbooster12-dev) [inline be/app/Models/Follower.php]: - what's this for? is this relationship? i don't understand :thinking:

File: be/database/migrations/2022_11_11_021215_create_activity_logs_table.php [hunk 2]
```diff
@@ -15,7 +15,7 @@ public function up()
     {
         Schema::create('activity_logs', function (Blueprint $table) {
             $table->id();
-            $table->string('loggable_id');
+            $table->string('loggale_id');
```
Comment (escbooster12-dev) [inline be/database/migrations/2022_11_11_021215_create_activity_logs_table.php]: - undo this changes.

File: fe/src/pages/authentication/BufferPage.jsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import React, { useEffect } from "react";
```
Comment (escbooster12-dev) [inline fe/src/pages/authentication/BufferPage.jsx]: - again BufferPage

File: be/app/Models/Follower.php (hunk 1 above)
Comment (ragg09-sunasterisk) [reply be/app/Models/Follower.php]: I am just checkint out how it will word I can remove it po

**Review by escbooster12-dev — APPROVED** [review]
